bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/Capability.java

```diff
@@ -104,15 +104,14 @@ public class Capability {
 
     protected void beforeNewData() {
         properties = new HashMap<>(thing.getProperties());
-        firstLaunch = properties.isEmpty();
-        if (firstLaunch) {
-            properties.put(PROPERTY_THING_TYPE_VERSION, moduleType.thingTypeVersion);
-            if (!moduleType.isLogical()) {
-                String name = moduleType.apiName.isBlank() ? moduleType.name() : moduleType.apiName;
-                properties.put(PROPERTY_MODEL_ID, name);
-                properties.put(PROPERTY_VENDOR, VENDOR);
-            }
+
+        // only the thingTypeVersion is present
+        firstLaunch = properties.size() <= 1;
```

Comment:
You are no more setting the thing type version ?

Reply:
Because it is already and also done in the ThingTypeProvider (line 79). I checked, it was duplicated action.

Reply:
Won't this break if another system property is added later? Is it correct that this check is an optimization to avoid updating the properties more than once (but actually there would be no harm in doing so)? If true, wouldn't it be possible to simply store this status in a boolean attribute? Like here:

https://github.com/openhab/openhab-addons/blob/035f4463e530d6374fd1a6bb6c4361de20438307/bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/handler/DanfossAirUnitHandler.java#L153-L178

Reply:
Yes, you're right, it is done to avoid redoing some thing modifications (e.g. home thing removes unemployed channels) and you're correct this is not bullet proof the way it is done.
Going to change that.